src/core_ocean/shared/mpas_ocn_wetting_drying.F

```diff
@@ -0,0 +1,712 @@
+! Copyright (c) 2013,  Los Alamos National Security, LLC (LANS)
+! and the University Corporation for Atmospheric Research (UCAR).
+!
+! Unless noted otherwise source code is licensed under the BSD license.
+! Additional copyright and license information can be found in the LICENSE file
+! distributed with this code, or at http://mpas-dev.github.com/license.html
+!
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  ocn_vmix
+!
+!> \brief MPAS ocean wetting and drying
+!> \author Phillip J. Wolfram
+!> \date   03/09/2018
+!> \details
+!>  This module contains the wetting and drying algorithms
+!>  to prevent cell thicknesses from becoming too small.
+!>
+!
+!-----------------------------------------------------------------------
+
+module ocn_wetting_drying
+
+   use mpas_kind_types
+   use mpas_derived_types
+   use mpas_pool_routines
+   use mpas_timer
+   use mpas_io_units
+
+   use ocn_constants
+   use ocn_diagnostics
+   use ocn_gm
+
+   implicit none
+   private
+   save
+
+   !--------------------------------------------------------------------
+   !
+   ! Public parameters
+   !
+   !--------------------------------------------------------------------
+
+   !--------------------------------------------------------------------
+   !
+   ! Public member functions
+   !
+   !--------------------------------------------------------------------
+
+   public :: ocn_wetting_drying_verify, ocn_prevent_drying_rk4, &
+     ocn_prevent_drying_split_explicit, &
```

Comment:
@mark-petersen, this is a stub for the split explicit implementation, is it ok to leave it in for now or do you want me to remove it?